.travis.yml

```diff
@@ -6,7 +6,7 @@ env:
 
 # keep this blank to make sure there are no before_install steps
 before_install:
-
+  npm install codecov
```

Comment:
it does, it's just my favor of avoiding install it in local machines for dev, only for CI machines. Since travis caches dependencies anyway, it doesn't hurt build speed much either.